### PR TITLE
feat(event): add hide_attendees

### PR DESCRIPTION
**Pull request by chillfox**

### CODE REVIEW

File: src/google.cr [hunk 1]
```diff
@@ -661,6 +661,8 @@ class Google::Calendar::Event
         organizer: attendee.organizer)
     end
 
+    hide_attendees = @guests_can_see_other_guests.nil? ? true : @guests_can_see_other_guests
```
Comment (stakach) [inline src/google.cr]: Shouldn't this be inversed, i.e.
`guests_can_see_other_guests = true` then this should be `hide_attendees = false`
I assume the default is `true` so if it's `nil` then `hide_attendees = false`

so the code should be:
```suggestion
    hide_attendees = @guests_can_see_other_guests.nil? ? false : !@guests_can_see_other_guests
```

File: src/office365.cr [hunk 2]
```diff
@@ -538,6 +538,7 @@ class Office365::Event
       title: @subject,
       body: @body.try &.content,
       attendees: attendees,
+      hide_attendees: @hide_attendees,
```
Comment (stakach) [inline src/office365.cr:542]: This file also needs the mapping going the other way
place event => office365 event

Need to update the `private def event_params(event)` function

File: src/google.cr [hunk 3]
```diff
@@ -689,6 +691,7 @@ class Google::Calendar::Event
       body: @description,
       location: @location,
       attendees: attendees,
+      hide_attendees: hide_attendees,
```
Comment (stakach) [inline src/google.cr:695]: This file also needs the place event => Google event
Need to update the `private def event_params(event, calendar_id)` function with reverse mapping

**Review by stakach — CHANGES_REQUESTED** [review]

File: src/office365.cr (hunk 2 above)
Comment (chillfox) [reply src/office365.cr:542]: ah, I see. I was looking for a `from_` or `to_` for the other way.

**Review by stakach — APPROVED** [review]
LGTM

**Pull request merged**